src/Modals/ModalNewForm/ModalNewForm.js

```diff
@@ -23,10 +24,15 @@ class PostForm extends Component {
 
 
     onChangeInput(type, answer){
-        this.setState({
-            [type]: answer,
-        })
-
+        if(type === 'images'){
```

Comment:
How about instead of creating a new special case, you just call `onChangeInput` with inputs that would **just work**?

e.g.,

`onChangeInput('image', this.state.images + answer)`